dotnet/aspnet46/webapp/Application/SampleWebApplication.FunctionalTests/SampleFunctionalTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Remote;
+using System;
+using System.IO;
+
+namespace SampleWebApplication.FunctionalTests
+{
+    [TestClass]
+    public class SampleFunctionalTests
+    {
+        private static TestContext testContext;
+        private RemoteWebDriver driver;
+        [ClassInitialize]
+        public static void Initialize(TestContext testContext)
+        {
+            SampleFunctionalTests.testContext = testContext;
+        }
+
+        [TestMethod]
+        public void SampleFunctionalTest1()
+        {
+            driver = GetChromeDriver();
+            driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(180);
+            var webAppUrl = testContext.Properties["webAppUrl"].ToString();
+            driver.Navigate().GoToUrl(webAppUrl);
+            Assert.AreEqual(driver.Title, "Home Page - My ASP.NET Application", "Expected title to be 'Home Page - My ASP.NET Application'");
+            var filePath = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString()) + ".png";
+            var screenshot = driver.GetScreenshot();
+            screenshot.SaveAsFile(filePath);
+            testContext.AddResultFile(filePath);
+        }
+
+        [TestCleanup]
+        public void TestCleanUp()
+        {
+            driver.Quit();
+        }
+
+        private RemoteWebDriver GetChromeDriver()
+        {
+            var path = Environment.GetEnvironmentVariable("ChromeWebDriver");
```

Comment:
Does this work on hosted?

Reply:
yes. This was the solution for hosted. We baked this env variable into the hosted images